pallets/connectors/Cargo.toml

```diff
@@ -40,7 +40,7 @@ runtime-common = { path = "../../runtime/common", default-features = false }
 # Polkadot
 xcm = { git = "https://github.com/paritytech/polkadot", default-features = false, branch = "release-v0.9.32" }
 
-pallet-ethereum = { git = "https://github.com/PureStake/frontier", default-features = false, branch = "moonbeam-polkadot-v0.9.32" }
```

Comment:
Why the change?

Reply:
Unfortunately, I had to point our `pallet-ethereum` dep from the `PureStake/frontier` fork to one newer revision as their `moonbeam-polkadot-v0.9.32` branch is missing one commit for the the `try-runtime` fixes. Now that they have [merged the hotfix](https://github.com/PureStake/frontier/pull/162#event-8657194321) into that branch, I can revert this change.